integration-tests/bean/src/main/java/org/apache/camel/quarkus/component/bean/method/EmployeeServiceBean.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.bean.method;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Named;
+
+import io.quarkus.arc.Unremovable;
+import org.apache.camel.quarkus.component.bean.model.Employee;
+
+@ApplicationScoped
+@Named("EmployeeServiceBean")
+@Unremovable
```

Comment:
Strange, I would have thought that this class would need @RegisterForReflection.

Reply:
It may be caught by native tests actually. I've seen `org.apache.camel.component.bean.MethodNotFoundException: Method with name: toFirstName not found on bean` on bean native tests.

Reply:
Yep, `@RegisterForReflection` is required for both method language and class component. I have sent this PR blindly thinking they are perhaps not required. Now I know they are :).